### PR TITLE
[300] Created unit test to  CategoryDtoResponseMapper

### PR DESCRIPTION
**Pull request by Maryna-511750**

### CODE REVIEW

File: service/src/test/java/greencity/mapping/CategoryDtoResponseMapperTest.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
+package greencity.mapping;
+
+import greencity.dto.category.CategoryDtoResponse;
+import org.junit.jupiter.api.Test;
+import greencity.entity.Category;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.modelmapper.ModelMapper;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (OlehTsipotan) [inline service/src/test/java/greencity/mapping/CategoryDtoResponseMapperTest.java]: organize imports

File: service/src/test/java/greencity/mapping/CategoryDtoResponseMapperTest.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+@ExtendWith(SpringExtension.class)
+class CategoryDtoResponseMapperTest {
+    private final ModelMapper modelMapper = new ModelMapper();
+    private final CategoryDtoResponseMapper categoryDtoResponseMapper = new CategoryDtoResponseMapper();
+    @Test
+    void convert_CategoryDtoResponseMapperTest_ShouldMapCorrectly() {
+        Category category = new Category(55L,"Name", "Ім'я", null, null);
```
Comment (OlehTsipotan) [inline service/src/test/java/greencity/mapping/CategoryDtoResponseMapperTest.java]: Add a line between the class declaration and fields.
Add a line between a field and a method.

File: service/src/test/java/greencity/mapping/CategoryDtoResponseMapperTest.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+    @Test
+    void convert_CategoryDtoResponseMapperTest_ShouldMapToCategoryWithNullNameAndNullId() {
+        Category category = new Category();
+
+        CategoryDtoResponse actual = categoryDtoResponseMapper.convert(category);
+
+        assertNotNull(actual);
+        assertEquals(null, actual.getId());
+        assertEquals(null, actual.getName());
```
Comment (OlehTsipotan) [inline service/src/test/java/greencity/mapping/CategoryDtoResponseMapperTest.java]: try `asseertNull`

**Review by OlehTsipotan — APPROVED** [review]
Well-organized and easy-to-read code with just a few spaces to improve

**Review by ryhaldenys — APPROVED** [review]

**Pull request merged**